config/sync/field.storage.node.field_datasets.yml

```diff
@@ -0,0 +1,19 @@
+uuid: cff58109-7a29-46f1-902f-e2962787df8e
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_datasets
+field_name: field_datasets
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node
+module: core
+locked: false
+cardinality: 4
```

Comment:
Where did the limit of 4 datasets come from?

Reply:
Arbitrary number, The logic being a Performance measure could use more than one data source but no one specified what the limit should be, so i put it as 4 until i get some feedback on what should it be.

Reply:
Leave it as unlimited references.  I don't see a point in any number anyone could come up with.